docs/builtin_workflows.rst

```diff
@@ -510,6 +510,180 @@ the model-fitting workflows and which sampling schemes work with them.
 
 \* Not recommended
 
+.. _connectivity_matrices:
+
+#####################
+Connectivity matrices
+#####################
```

Comment:
```suggestion
*********************
Connectivity matrices
*********************
```

The section labeling needs to be adjusted. Here's the convention we have in conf.py.

````
# with overline, for parts
* with overline, for chapters
= for sections
- for subsections
^ for subsubsections
" for paragraphs
```